pkg/proxy/envvar_test.go

```diff
@@ -0,0 +1,42 @@
+package proxy_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/operator-framework/operator-marketplace/pkg/proxy"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestGetOperatorEnvVars(t *testing.T) {
```

Comment:
Would this test pass even if `TestSetEnvVars()` ran first?

Reply:
Good point. I will set the env vars above.